docs/section11/Rule11-6.md

```diff
@@ -0,0 +1,38 @@
+# HVAC_SystemZoneAssignment - Rule 11-6
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-6  
+**Rule Description:** Piping losses shall not be modeled. 
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, i
+
+**Evaluation Context:** Each SWH BAT  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_associated_with_each_SWH_bat()
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are SHW loads
+- also use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW BATs and SHW use in the building: `shw_bats_and_use_dict = get_SHW_equipment_associated_with_each_SWH_bat(P_RMD)`
+- look through each of the SHW BATs in the building: `for shw_bat in shw_bats_and_use_dict:`
+  - check if there are SHW loads in this use type, continue to rule logic: `if len(shw_bats_and_use_dict[shw_bat]) > 0: RULE_APPLICABLE`
```

Comment:
Not clear why ServiceWaterHeatingUse is required for this to be applicable. What is the scenario that this prevents? Couldn't `ServiceWaterPiping.piping_losses_modeled` be defined as true without ServiceWaterHeatingUse being defined?